### PR TITLE
add jekyll to the olm whitepaper

### PR DESCRIPTION
**Pull request by acabunoc**
Hey @chadsansing!

Formatting is not great, but you can see it here: https://acabunoc.github.io/olm-whitepaper/
(haven't added headings to the sidebar yet)

We can chat later if you'd like to know what each of the files do! I'll be adding commits to fix formatting and such. Leave a comment in this PR if you want anything specific. You should also be able to push to this branch if you'd like to try out your own cool stylezzz

### CODE REVIEW

**Comment by acabunoc** [comment]
By chat later, I mean chat in January

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+title: Open Leadership Training Series
+subtitle:
+email: abby@mozillafoundation.org
```
Comment (acabunoc) [inline _config.yml]: I left my email & the OL twitter and chat stuff. Please feel free to update

File: _articles/whitepaper.md [hunk 2]
```diff
@@ -1,28 +1,12 @@
-# Open Leadership Map Whitepaper v.0.4 (public alpha)
+---
+layout: page
+title: "White Paper"
```
Comment (acabunoc) [inline _articles/whitepaper.md]: Question: is it "whitepaper" or "white paper"?

Comment (chadsansing) [reply _articles/whitepaper.md]: We should switch to "white paper" throughout, I think. Good catch.

**Pull request merged**